### PR TITLE
Update `Beatmap/Break` with extra details and screenshots

### PR DESCRIPTION
**Pull request by WindowsMeosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Gameplay/Pass/de.md [hunk 1]
```diff
@@ -7,4 +7,4 @@ tags:
 
 # Pass
 
-Ein **Pass** beschreibt die Absolvierung einer [Beatmap](/wiki/Beatmap) oder einer bestimmten Beatmap-Sektion ohne die Benutzung von [Mods](/wiki/Gameplay/Game_modifier), die ein Scheitern verhindern (z. B. [No Fail](/wiki/Gameplay/Game_modifier/No_Fail)), oder das Erreichen eines Pausenabschnitts mit mehr als 50% [Leben](/wiki/Gameplay/Health). Sofern ein Pausenabschnitt erreicht wird, zeigt ein Indikator mit Sound an, dass der Spieler den vorherigen Abschnitt absolviert hat.
+Ein **Pass** beschreibt die Absolvierung einer [Beatmap](/wiki/Beatmap) oder einer bestimmten Beatmap-Sektion ohne die Benutzung von [Mods](/wiki/Gameplay/Game_modifier), die ein Scheitern verhindern (z. B. [No Fail](/wiki/Gameplay/Game_modifier/No_Fail)), oder das Erreichen eines [Pause](/wiki/Beatmap/Break)nabschnitts mit mehr als 50% [Leben](/wiki/Gameplay/Health). Sofern ein Pausenabschnitt erreicht wird, zeigt ein Indikator mit Sound an, dass der Spieler den vorherigen Abschnitt absolviert hat.
```
Comment (The-Last-Cookie) [inline wiki/Gameplay/Pass/de.md:10]:
```suggestion
Ein **Pass** beschreibt die Absolvierung einer [Beatmap](/wiki/Beatmap) oder einer bestimmten Beatmap-Sektion ohne die Benutzung von [Mods](/wiki/Gameplay/Game_modifier), die ein Scheitern verhindern (z. B. [No Fail](/wiki/Gameplay/Game_modifier/No_Fail)), oder das Erreichen eines [Pausenabschnitts](/wiki/Beatmap/Break) mit mehr als 50% [Leben](/wiki/Gameplay/Health). Sofern ein Pausenabschnitt erreicht wird, zeigt ein Indikator mit Sound an, dass der Spieler den vorherigen Abschnitt absolviert hat.
```

File: wiki/Beatmap/Break/fr.md [hunk 2]
```diff
@@ -7,10 +7,14 @@ tags:
...
+outdated_translation: true
 ---
 
 # Pause
 
 Les **pauses** sont des sections d'une [beatmap](/wiki/Beatmap) qui n'ont aucun [objet](/wiki/Gameplay/Hit_object). Pendant ce temps, le [drain de vie](/wiki/Gameplay/Health) est temporairement interrompu. Les pauses permettent généralement au joueur de reposer son bras pendant un court instant et de repositionner son [dispositif d'entrée](/wiki/Gameplay/Input_device).
 
 Selon les [paramètres](/wiki/Client/Beatmap_editor/Song_setup) activés, des letterboxes peuvent apparaître lorsqu'une pause est initiée. En outre, un graphique et un son de réussite ou d'échec peuvent apparaître pour indiquer si le joueur s'en sort bien ou non. Cela n'apparaîtra que s'il y a suffisamment de temps pour que l'animation s'affiche.
+
+![Capture d'écran d'une passe de section de pause](img/Section_pass.jpg "Passe de section")
```
Comment (DoryanR) [inline wiki/Beatmap/Break/fr.md:20]:
```suggestion
![Capture d'écran d'une section réussi sur l'écran de pause](img/Section_pass.jpg "Section réussi") ![Capture d'écran d'un section miss sur l'écran de pause](img/Section_fail.jpg "Section miss")
```

**Review by DoryanR — CHANGES_REQUESTED** [review]
Changes to the French file. **The image `Section_fail.jpg` is missing for the translations**.

File: wiki/Beatmap/Break/fr.md (hunk 2 above)
Comment (WindowsMeosu) [reply wiki/Beatmap/Break/fr.md:20]: yeah that was intentional, I was sort of on a time limit.

**Comment by WindowsMeosu** [comment]
bump

**Review by TicClick — CHANGES_REQUESTED** [review]

File: wiki/Beatmap/Break/en.md [hunk 3]
```diff
@@ -10,4 +10,6 @@ tags:
 
 **Breaks** are sections of a [beatmap](/wiki/Beatmap) which have no [hit objects](/wiki/Gameplay/Hit_object). During this time, [health drain](/wiki/Gameplay/Health) is temporarily halted. Breaks usually allow the player to rest their arm for a short while and reposition their [input device](/wiki/Gameplay/Input_device).
 
-Depending on the enabled [settings](/wiki/Client/Beatmap_editor/Song_setup), letterboxes may appear when a break is initiated. In addition to this, a passing or failing graphic and sound may appear to indicate if a player is doing well or not. This will only appear if there is enough time for the graphic to show.
+Depending on the enabled [settings](/wiki/Client/Beatmap_editor/Song_setup), letterboxes may appear when a break is initiated. In addition to this, a passing or failing graphic and sound may appear to indicate if a player is doing well or not. This will only appear if there is enough time for the graphic to show, and will not appear if the mode is osu!mania. The player's current [grade](/wiki/Gameplay/Grade) may be shown at the top right corner of the screen.
```
[the remark this answers is not in the document]
Comment (TicClick) [reply wiki/Beatmap/Break/en.md:13]: (also if applied, this unfortunately tosses localisation edits into the bin)

[the remark this answers is not in the document]
Comment (WindowsMeosu) [reply wiki/Beatmap/Break/en.md]: > (also if applied, this unfortunately tosses localisation edits into the bin)

In essence, the only localization that was wasn't effortlessly worked on was the indonesian ones. I could probably just apply some edits and ask wowcake to review again

**Review by cl8n — DISMISSED** [review]

**Comment by cl8n** [comment]
please don't include changes to translations in the same pr, it makes reviewing difficult and forces translations to be scrapped if anything changes further (as it has here)

**Review by cl8n — APPROVED** [review]

**Review by TicClick — DISMISSED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**